### ISSUE TITLE
"schannel: failed to retrieve ALPN result" when running on Windows 7

### ISSUE THREAD

**Issue by attibalazs** [sentinelsat/sentinelsat#58]
I`m running into issues on Windows 7, did anyone else encounter this problem ?

```
  File "C:\Users\myuser\AppData\Local\Continuum\Anaconda3\lib\site-packages\homura.py", line 167, in curl
    c.perform()
pycurl.error: (35, 'schannel: failed to retrieve ALPN result')
```

Might be an issue related to curl:
https://github.com/curl/curl/issues/840

Looks like this is fixed in the latest version of curl 7.50:
curl/curl@34855fe

**Comment by attibalazs** [comment]
This is my workaround in sentinel.py until you guys sort it.

```
        # homura.download(product_info['url'], path=path,session=self.session, **kwargs)
        # workaround since homura works with pycurl which has issues on windows 7
        with open(path, 'wb') as handle:
            response = requests.get(product_info['url'], stream=True, auth=self.session.auth)
            if not response.ok:
                print(response.text)
            else:
                for block in response.iter_content(1024):
                    handle.write(block)
```

**Comment by Fernerkundung** [comment]
Fixed in [v0.11](https://github.com/sentinelsat/sentinelsat/releases/tag/v0.11) release, which removed the `curl` dependency.
